src/api/Feed.js

```diff
@@ -167,9 +173,10 @@ class Feed extends Base {
         this.errorCallback = onError;
         const versionsPromise = this.fetchVersions();
         const commentsPromise = this.fetchComments(permissions);
+        const appActivityPromise = this.fetchAppActivity(permissions);
         const tasksPromise = shouldShowNewTasks ? this.fetchTasksNew() : this.fetchTasks();
 
-        Promise.all([versionsPromise, commentsPromise, tasksPromise]).then(feedItems => {
+        Promise.all([versionsPromise, commentsPromise, tasksPromise, appActivityPromise]).then(feedItems => {
```

Comment:
Will the app activity items be rendered in a separate PR?

Reply:
This is the Render PR. Note that it's a branch on top of the API layer branch